src/commands/support/deadzones.ts

```diff
@@ -5,7 +5,7 @@ import { CommandCategory } from '../../constants';
 const DEADZONE_IMAGE_URL = 'https://cdn.discordapp.com/attachments/770835189419999262/802254518376464424/Deadzone_Controls.png';
 
 export const deadzones: CommandDefinition = {
-    name: ['deadzone', 'dz'],
+    name: ['deadzone', 'deadzones', 'dz'],
```

Comment:
```suggestion
    name: ['deadzones', 'deadzone', 'dz'],
```

Gonna be a pain here. `.deadzones` is the command name in docs, and the filename, so will just put that first here haha

Reply:
good shout!